include/nw4r/ut/Color.h

```diff
@@ -7,18 +7,31 @@ namespace ut {
 
 struct Color : public _GXColor {
```

Comment:
Worth noting: `typedef struct A {} B;` will show in DWARF as `A`.

It's possible that `_GXColor` is the struct tag, and is actually typedef'd to `GXColor`. Every GX typename seems to do this, and so OGWS uses the following convention:
```c
typedef struct _GXColor {
    u8 r, g, b, a;
} GXColor;
```

Then it would be `struct Color : public GXColor`.
Just preference though, do whatever you want lol

Reply:
This assumption is likely right, as it would seem GX interfaces do not use underscores. Implemented.